frontend/app/mocks/cct-api.server.ts

```diff
@@ -21,10 +21,17 @@ export function getCCTApiMockHandlers() {
     //
     // Handler for GET requests to retrieve pdf
     //
-    http.get('https://api.example.com/cct/letters/:referenceId', async ({ params, request }) => {
+    http.get('https://api.example.com/cctws/OnDemand/api/GetPdfByLetterId?userid=userId&community=community&id=referenceId', async ({ request }) => {
       log.debug('Handling request for [%s]', request.url);
 
-      const pdfEntity = getPdfEntity(params.referenceId);
+      const url = new URL(request.url);
+      const community = url.searchParams.get('community');
+      const userId = url.searchParams.get('userid');
+      const referenceId = url.searchParams.get('id');
+
+      if (community === null || userId === null || referenceId === null) throw new Error('Query parameters must be defined');
```

Comment:
[Read a single parameter](https://mswjs.io/docs/recipes/query-parameters/#read-a-single-parameter) then return a 400 if params are not valid.

```suggestion
      if (community === null || userId === null || referenceId === null) {
        return new HttpResponse(null, { status: 400 })
      }
```

Reply:
updated